test/src/analyzers/lint_analyzer/rules/rules_list/ban_name/examples/example.dart

```diff
@@ -15,3 +16,13 @@ void strangeName() {} // LINT
 class AnotherStrangeName {
   late var strangeName; // LINT
 }
+
+StrangeClass.someMethod(); // LINT
+NonStrangeClass.someMethod();
+
+DateTime.now(); // LINT
+DateTime.now().day; // LINT
+class DateTimeTest {
+  final currentTimestamp = DateTime.now(); // LINT
+}
+DateTime now = DateTime(2022);
```

Comment:
Please wrap all the statements inside a main. I'm not sure that this code is parsed properly.